scss/vendor/_rfs.scss

```diff
@@ -0,0 +1,204 @@
+// stylelint-disable property-blacklist, scss/dollar-variable-default
+
+// SCSS RFS mixin
+//
+// Automated font-resizing
+//
+// See https://github.com/twbs/rfs
+
+// Configuration
+
+// Minimum font size
+$rfs-minimum-font-size: 1rem !default;
```

Comment:
I'd recommend change `$rfs-minimum-font-size` to `1.25rem` (same the`$font-size-lg`). Because `$rfs-minimum-font-size: 1rem` make a change in font size of large sized UI controls such as `.btn-lg` with responsive.

```css
.btn-lg, .btn-group-lg > .btn {
  ...
  font-size: 1.25rem;
  ...
}

@media (max-width: 1200px) {
  .btn-lg, .btn-group-lg > .btn {
    font-size: calc(1.05rem + 0.266667vw) ;
  }
}
```

But perhaps many developers can not expect it.

If `$rfs-minimum-font-size` is `1.25rem`, RFS create a responsive font size to only `legend`, `h1` to `h4`, `.display-x` and `.close` in default. (IMO `.close` don't need have responsive font size)

Any thoughts?

Reply:
Thanks for the review @ysds! I chose `1rem` because:
- It's the same as the base font size
- This is about the lowest font size that's always readable
- It's an easy to remember base
- This font size will less likely cause rescaling issues with display headings

About the `.close` font size, I think this is another issue. The current implementation doesn't cause any issues. We 'll probably need to review this in the future, because at the moment this element is aligned by line-height and padding and it's hard to get perfect control over it. But I don't yet want to change something here to prevent unwanted results in `v4`. 
![image](https://user-images.githubusercontent.com/11559216/48829598-ecd16280-ed72-11e8-8287-9f9c33d8a50e.png)

Reply:
My concern is who want to change font size of large controls (e.g. .btn-lg) in responsive. Users want just a responsive font size of a large content element like a heading IMO.

Reply:
I understand, I'm going to test what effect this change has. I've added it to the todo list.